server/routes/oodlet.js

```diff
@@ -70,33 +71,74 @@ let routes = [
   {
     method: 'GET',
     path: '/oodlet',
-    handler: list
+    config: {
+      handler: list,
+      validate: {
+        query: {
+          office: Joi.string().required(),
+          toDate: Joi.date(),
+          fromDate: Joi.date()
+        }
+      }
+    }
   },
   {
     method: 'GET',
     path: '/oodlet/{id}',
-    handler: get
+    config: {
+      handler: get,
+      validate: {
+        params: {
+          id: Joi.string().required()
+        }
+      }
+    }
   },
   {
     method: 'POST',
     path: '/oodlet',
-    handler: create
+    config: {
+      handler: create,
+      validate: {
+        payload: {
+          oodlerId: Joi.string().required(),
+          quantifiedThingies: Joi.array()
```

Comment:
I would like to merge #38 first so I can do Joi validation on db models and then use that Thingy scheme for this validation. This was WIP version, anyway, since I had in mind what I just stated.